platform/platform-app/src/main/resources/features.xml

```diff
@@ -760,6 +760,17 @@
         <bundle>mvn:ddf.platform.migration/platform-migration/${project.version}</bundle>
     </feature>
 
+    <feature name="platform-uuidgenerator-api" install="manual" version="${project.version}"
```

Comment:
Should feature naming match the module naming and include `util` in the name (`platform-util`)?